kratos/templates/template_application/TemplateApplication.py.in

```diff
@@ -1,7 +1,13 @@
 # Application dependent names and paths
-from KratosMultiphysics import _ImportApplication
+from KratosMultiphysics import _ImportApplication, python_registry_utilities
 from Kratos@{APP_NAME_CAMEL}Application import *
+
 application = Kratos@{APP_NAME_CAMEL}Application()
 application_name = "Kratos@{APP_NAME_CAMEL}Application"
 
 _ImportApplication(application, application_name)
+
+from . import python_registry_lists
+python_registry_utilities.RegisterModelersList("KratosMultiphysics.@{APP_NAME_CAMEL}Application", python_registry_lists)
+python_registry_utilities.RegisterOperationsList("KratosMultiphysics.@{APP_NAME_CAMEL}Application", python_registry_lists)
+python_registry_utilities.RegisterProcessesList("KratosMultiphysics.@{APP_NAME_CAMEL}Application", python_registry_lists)
```

Comment:
```suggestion
python_registry_utilities.RegisterProcessesKratosMultiphysics.@{APP_NAME_CAMEL}Application", python_registry
```

more pythonic

Reply:
You mean passing the module instead? If so, I've two concerns:
1. We already have a `python_registry` module
2. In the long run, the idea would be to substitute the `python_registry_lists` by json ones stored in the application.json files. If we do it in this way we won't require to change the implementations in the `python_registry_utilities.py`.